llvm/lib/ObjectYAML/MinidumpYAML.cpp

```diff
@@ -550,7 +588,7 @@ Stream::create(const Directory &StreamDesc, const object::MinidumpFile &File) {
   llvm_unreachable("Unhandled stream kind!");
 }
 
-Expected<Object> Object::create(const object::MinidumpFile &File) {
+Expected<Object> Object::create(object::MinidumpFile &File) {
```

Comment:
@labath I had to drop the const modifier here in order to cache the MemoryDescriptor_64 offsets. I thought this would be appropriate given repetitive linear search in the `Stream::create` for Memory64. I'm not sure if removing the const modifier is the correct decision and would appreciate your input.

Reply:
`fallible_iterator` is really cool! I'll refactor everything over